docs/operator-guides/using-custom-connectors.md

```diff
@@ -1,4 +1,4 @@
-# Using custom connectors
+# Uploading Docker-based custom connectors
```

Comment:
This is now pretty long within the actual sidebar. Do we maybe still want to keep a shorter title for the sidebar?

You could do that easily by putting the following on top of this document:

```mdx
---
sidebar_label: Shorter label
---
```